tests/spec/core/utils-spec.js

```diff
@@ -317,16 +317,11 @@ describe("Core - Utils", () => {
       expect(intlEn.foo).toBe("EN Foo");
     });
 
-    it("falls back to English string if key does not exit in language", () => {
+    it("falls back to English string if key does not exist in language", () => {
       const intl = getIntlData(localizationStrings, "ko");
       expect(intl.bar).toBe("EN Bar");
     });
 
-    it("falls back to English string if language does not exit in language data", () => {
```

Comment:
Need this test.

Reply:
Ah, my fault! Sorry! I mis-interpreted the test as duplication of the one before...

Reply:
Re-added the test in https://github.com/w3c/respec/pull/2687/commits/ea2f39a7ce6ff10fa81eb7868f0c522838b3ea7a